server/services/store/sqlstore/blocks.go

```diff
@@ -42,27 +43,31 @@ func (s *SQLStore) timestampToCharField(name string, as string) string {
 	}
 }
 
-func (s *SQLStore) blockFields() []string {
+func (s *SQLStore) blockFields(alias string) []string {
```

Comment:
nit: we have a similar but slightly different mechanism on the `boards` store, I'd suggest to pick one and use it on everywhere for consistency.

As for which one to choose, I'm leaning more towards the one proposed on this PR

Reply:
Thanks for reviewing @mgdelacroix.  I modified the boards store to be consistent.